terraform/workspace-variables/dev.tfvars

```diff
@@ -11,4 +11,4 @@ paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
 paas_web_app_memory = 1024
-paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && /app/bin/delayed_job --pool=mailers --pool=*:2 start && rails db:safe_reset && rails cron:schedule && rails s"
+paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate s3:delete_csv && /app/bin/delayed_job --pool=mailers --pool=*:2 start && rails db:safe_reset && rails cron:schedule && rails s"
```

Comment:
You need an extra `s` at the end of the command name - `delete_csv` vs `delete_csvs` here and in review tfvars file.

Reply:
Yep, spotted that, thanks!